Makefile

```diff
@@ -1,7 +1,7 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= goharbor/harbor-operator:dev
-RELEASE_VERSION ?= 0.0.0-dev
+IMG ?= goharbor/harbor-operator:v1.0.0
```

Comment:
Do we have an image with the tag `v1.0.0` now?

Reply:
no, there's no image with tag v1.0.0 now, only v1.0.0-rc1
but I think anyway it would be better than tag dev